src/sentry/api/bases/organization.py

```diff
@@ -177,9 +182,12 @@ def get_projects(
         :param include_all_accessible: Whether to factor the organization
         allow_joinleave flag into permission checks. We should ideally
         standardize how this is used and remove this parameter.
+        :param project_ids: Projects if they were passed via request
+        data instead of get params
         :return: A list of Project objects, or raises PermissionDenied.
         """
-        project_ids = self.get_requested_project_ids(request)
+        if project_ids is None:
+            project_ids = self.get_requested_project_ids(request)
```

Comment:
Instead of extra parameters should this method look in GET first, and then fallback to POST data if the request isn't a GET?

Reply:
Hmm I was thinking this way we'd avoid the potential issues of different POST/PUT requests using different keys to store project_ids eg `project`, `projects`, `project_ids` etc.

Reply:
I would hope we don't use a variety of parameter names in the API, but it is also very possible. Having the new behavior be opt-in through the use of more parameters will be safer.